process-engine/process_engine_runtime
# ♻️ Move post-migrations to internal module

## Changes

1. Move top level modules to `modules` subfolder (except for 'main.ts', which is the entry point for the application)
2. Move commonly used environmental operations to `environment.ts`
3. Move post-migrations from npm scripts into internal modules
    - This removes the need for duplicated environment initialization
    - It also fixes an issue when running the post-migration from within an `app.asar`, where the runtime's npm scripts are unavailable per default
4. Always run post-migrations during startup, after the sequelize-migrations are done
5. Add some rule overrides to `.eslintignore`
6. Disable `npm-install-only` Step during CI builds, since this has frequently caused issues with Jenkins' npm cache

## Issues

Closes #452 

PR: #453

## How to test the changes

Startup the runtime and notice how Post-Migrations are automatically run after the migrations.
